compiler/rustc_macros/src/query.rs

```diff
@@ -114,6 +114,9 @@ struct QueryModifiers {
 
     /// Always remap the ParamEnv's constness before hashing.
     remap_env_constness: Option<Ident>,
+
+    /// Generate a `feed` method to set the query's value from another query.
+    feedable: Option<Ident>,
```

Comment:
We may want to do this unconditionally in the future, or is there any downside to it beyond bloat?

Reply:
ah, saw it later, this obviously conflicts with `no_hash` and `eval_always`